### PR TITLE
English phrasing corrections + react-electron dev tools 

### PR DESCRIPTION
**Pull request by James-E-Adams**
I've added and provided instructions for https://github.com/firejune/electron-react-devtools. Should make life a bit easier.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -94,6 +94,12 @@ yarn install
 yarn dev
 ```
 
+To use react dev tools, you can use [electron-react-devtools](https://github.com/firejune/electron-react-devtools), which should already be installed. When you open devtools, type the following into the console:
+```javascript
+require('electron-react-devtools').install()
```
Comment (Raathigesh) [inline README.md]: Should we be doing this from the main.ts script when we are in dev mode?

File: README.md [hunk 2]
```diff
@@ -94,6 +94,12 @@ yarn install
...
+```
+
+And then simply refresh (F5). You'll now see a React tab in dev tools. Happy debugging!
```
Comment (Raathigesh) [inline README.md]: Sorry for nitpicking but an empty line after here would be nice? 😄

Comment (James-E-Adams) [reply README.md]: I'm all for nitpicking! New line added.

File: README.md (hunk 1 above)
Comment (James-E-Adams) [reply README.md]: Let me look into this. :)

Comment (James-E-Adams) [reply README.md]: Did you mean the index file? I've tried there but it must be called from the renderer process, so investigating now if there's somewhere else I can call install.

Comment (James-E-Adams) [reply README.md]: Ahh - it's not even working sending a message as we did earlier :P

Comment (Raathigesh) [reply README.md]: Let's get it in and see for now.

**Pull request merged**